src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -275,6 +278,12 @@ class boss_gormok : public CreatureScript
 
                 DoMeleeAttackIfReady();
             }
+
+            void PassengerBoarded(Unit* who, int8 seatId, bool apply) override
+            {
+                if (!apply && seatId == 4) // hand
+                    who->ToCreature()->AI()->DoAction(ACTION_JUMP_ON_PLAYER);
```

Comment:
the only random shit is not to properly check an object type before static_cast<>'ing it